src/theme/DocItem/Footer/index.css

```diff
@@ -0,0 +1,3 @@
+:root {
+  --feedback-primary-color: #fc5806;
+}
```

Comment:
```suggestion
}

.button--icon {
  line-height: 1.0em;
}
```

non-blocking, nit (part 1): 

The icons in the buttons look uncentered vertically to me. There are two things going on: 

1. The buttons have a `line-height: 1.5` set on them that creates a few extra pixels of baseline beneath the icons.
2. The thumbs-down button has a visual weight thing where, even though it is in line with the thumbs up button, it _looks_ like there's more space beneath it. 

Adding this style, then incorporating it into the Footer/index.js component (see parts 2 and 3), would resolve number 1 by removing the extra baseline.

Here's a comparison of the buttons currently, vs with this line-height change: 

| without my proposed changes | with my proposed changes |
| --- | --- |
| ![without changes](https://github.com/user-attachments/assets/b41cde12-f4e8-45c1-ad64-0164f15dd321) | ![with changes](https://github.com/user-attachments/assets/5c6d0da1-0fee-49c5-bd46-4ac180e43483) |

I don't think there's anything for us to do about number 2, my brain needs to just accept the difference in visual weight.